fruit_classifier/utils/file_utils.py

```diff
@@ -23,13 +24,35 @@ def copytree(src, dst, symlinks=False, ignore=None):
     https://stackoverflow.com/questions/1868714/how-do-i-copy-an-entire-directory-of-files-into-an-existing-directory-using-pyth
     """
 
-    src = str(src)
-    dst = str(dst)
+    src = Path(src)
+    dst = Path(dst)
 
     for item in tqdm(os.listdir(src), desc='Copying files'):
-        s = os.path.join(src, item)
-        d = os.path.join(dst, item)
-        if os.path.isdir(s):
-            shutil.copytree(s, d, symlinks, ignore)
+        s = src.joinpath(item)
+        d = dst.joinpath(item)
+
+        if s.is_dir() and not d.exists():
+            shutil.copytree(str(s), str(d), symlinks, ignore)
+        elif not d.exists():
+            shutil.copy2(str(s), str(d))
         else:
-            shutil.copy2(s, d)
+            pass
+
+    print(os.listdir(src))
+    print(os.listdir(dst))
+
+
+if __name__ == '__main__':
+    #Testing the copytree function
```

Comment:
Why are we adding a `__main__` here?
Are we expecting that someone would like to run `file_utils.py` as a script?

Reply:
*__Main__* was added here to test the function and see how it worked. I left it for future testing, but have removed it since it is already tested thoroughly.